themes/shijin4/layouts/partials/navbar.html

```diff
@@ -22,6 +22,7 @@
 				<li><a href="/community/">Community</a></li>
 				<li><a href="/development/">Development</a></li>
 				<li><a href="/documents/">Documents</a></li>
+				<li><a href="/community/getting-involved/">Get Involved!</a></li>
```

Comment:
Before doing this I would merge the "documents" and "development" pages (they largely list the same things anyway) so we avoid adding yet another entry to this menu.

Reply:
It would get too convoluted for the time being.

Reply:
What I'm saying is I would:

- Leave this as it is with just 4 entries for now
- Later: merge documents and development
- Finally: add get-involved here